### PR TITLE
Update Vagrantfile for routeros with new box

### PR DESCRIPTION
**Pull request by luqasz**

### CODE REVIEW

**Comment by luqasz** [comment]
Boxes can be downloaded from https://drive.google.com/open?id=0B7lhzJhvsd-8dFB0NnNFUm52cjA

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -70,12 +70,15 @@ Vagrant.configure(2) do |config|
...
-    ros.vm.box = "mikrotik-chr-6.36"
-
+    # Download from https://drive.google.com/open?id=0B7lhzJhvsd-8dFB0NnNFUm52cjA
+    ros.vm.box = "chr-6.37.4"
     ros.vm.network :forwarded_port, guest: 22, host: 12205, id: 'ssh'
-
-    ros.vm.network "private_network", virtualbox__intnet: "link_1", ip: "169.254.1.11", auto_config: false
-    ros.vm.network "private_network", virtualbox__intnet: "link_2", ip: "169.254.1.11", auto_config: false
+    ros.vm.network :forwarded_port, guest: 8728, host: 8728, id: 'api'
+    ros.vm.network :forwarded_port, guest: 8729, host: 8729, id: 'apis'
```
Comment (dbarrosop) [inline vagrant/Vagrantfile]: What is the difference between `api` and `apis`? SSL?

Comment (luqasz) [inline vagrant/Vagrantfile]: Yes. `api` is plain text and `apis` has SSL

Comment (mirceaulinic) [inline vagrant/Vagrantfile]: In that case can we just name it `apissl` to make it more obvious, please?

**Comment by mirceaulinic** [comment]
Thanks @luqasz

**Pull request merged**